### DISCUSSION TITLE
v2.4.6

### DISCUSSION THREAD

**Discussion by MAKOMO** [artisan-roaster-scope/artisan#651] [Versions]
This version adds __[tracking of energy consumption & CO₂ production](https://artisan-roasterscope.blogspot.com/2021/07/tracking-energy-consumption-co2.html)__

***
Release Sponsor: [Sweet Maria's]()

![](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/screenshots/sweet-marias-logo.jpeg?raw=true)

---

The Artisan project runs on donations from individuals and companies recognising the value of Artisan for their work or leisure. Consider supporting this project with your [donation](https://www.paypal.com/paypalme/MarkoLuther)!

---

__>>>  [Release Notes](https://artisan-roasterscope.blogspot.com/2021/07/artisan-v246.html) <<<__

See the [detailed release history](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/ReleaseHistory.md) for the complete list of additions, changes and fixes and the [Installation Instructions](https://github.com/artisan-roaster-scope/artisan/blob/master/wiki/Installation.md) for platform specific notes.

Builds are available for macOS 10.15, Windows 8, Linux glibc 2.18 and Raspbian Buster or newer.

---

__Licence Notes__

_Artisan is free software: you can redistribute it and/or modify
it under the terms of the GNU General Public License as published by
the Free Software Foundation, either version 3 of the License, or
(at your option) any later version._

_Artisan is distributed in the hope that it will be useful,
but WITHOUT ANY WARRANTY; without even the implied warranty of
MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. _

_Copies of the GNU General Public License has been included with this 
distribution in the file `LICENSE.txt`. An online version is available at
<http://www.gnu.org/licenses/>._

<hr /><em>This discussion was created from the release <a href='https://github.com/artisan-roaster-scope/artisan/releases/tag/v2.4.6'>v2.4.6</a>.</em>

**Comment by Nitesh212212** [comment]
Hi there. Can you tell which file to download to use the artisan in smartphone or a tablet?

**Comment by MAKOMO** [comment]
Artisan does not run on a smartphone nor tablet. See the list of supported platforms here:

https://artisan-scope.org/about/

**Comment by AlejoTuesta** [comment]
hi where can I find .dmg for Mac, I have v2.4.2. installed in my Mac and does not recognize the phidgets

**Reply by MAKOMO** [reply]
Binaries are only available for the last two versions. Why looking for outdated software? The was not a single Artisan version that does not recognize Phidgets (assuming the latest Phidget driver is correctly installed).

**Comment by raulhuamanayala** [comment]
Hola, ¿dónde puedo encontrar el archivo .deb para linux?. De la versión 3
0. 0. Solo en esa versión reconoce mi dispositivo

**Reply by MAKOMO** [reply]
Older Artisan releases are not supported and thus no longer available. Please use the latest Artisan release v3.1 which does support all devices supported by Artisan v3.0.0.

In case you find an issue with the current release, please file an issue with all the required details on your setup that are needed for us to resolve it.
